specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/AppResiliency.json

```diff
@@ -0,0 +1,190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerApps API Client",
+    "version": "2023-08-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{appName}/resiliencyPolicies/{name}": {
+      "put": {
+        "tags": [
+          "AppResiliency"
+        ],
+        "summary": "Create or update an application's resiliency policy.",
+        "description": "Create or update an application's resiliency policy.",
+        "operationId": "AppResiliency_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "appName",
+            "description": "Name of the Container App.",
+            "in": "path",
+            "required": true,
```

Comment:
define a pattern for the appName

Reply:
Do you have an example of this? No other API in this section of the spec appears to do this, including the parent.

https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2023-08-01-preview/specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/ContainerApps.json#L163

Reply:
Found an example pattern for the app name, added it in.